src/test/scala/it/pagopa/interop/purposemanagement/provider/PurposePersistentBehaviorSpec.scala

```diff
@@ -65,6 +66,27 @@ class PurposePersistentBehaviorSpec extends ScalaTestWithActorTestKit(SpecConfig
       result shouldBe expected
     }
 
+    "change state from Active to Rejected" in {
```

Comment:
not sure this test is very useful, because the `updatePurposeFromState` is not impacted by the rejection feature